src/main/java/com/ebay/myriad/scheduler/yarn/interceptor/YarnSchedulerInterceptor.java

```diff
@@ -22,10 +25,26 @@
     public void init(Configuration conf, AbstractYarnScheduler yarnScheduler) throws IOException;
 
     /**
-     * Invoked *after* {@link YarnScheduler#handle(Event)}
+     * Invoked *before* {@link RMNodeImpl#handle(RMNodeEvent)}.
      *
      * @param event
+     * @param context
      */
-    public void onEventHandled(SchedulerEvent event);
+    public void beforeRMNodeEventHandled(RMNodeEvent event, RMContext context);
+
+    /**
+     * Invoked *before* {@link YarnScheduler#handle(org.apache.hadoop.yarn.event.Event)}
+     * @param event
+     */
+    public void beforeSchedulerEventHandled(SchedulerEvent event);
+
+    /**
```

Comment:
My fault, should've made the comment at line 43, but the code is
41 /**
42
43 /**
...
*/